### DISCUSSION TITLE
K8s Pod Memory usage goes beyond the limit

### DISCUSSION THREAD

**Discussion by BLasan** [grafana/grafana#38418] [Observability]
I deployed some simple microservices written in golang. And I checked the resource wastage after deploying them without sending any request to the services. It shows that the memory has exceeded the allocated limit at early stage (before even getting a request).
Anyone could explain what has caused here? When using kubernetes metric server it's totally different.
![Screenshot from 2021-08-22 10-21-53](https://user-images.githubusercontent.com/43112139/130343058-1bb3e018-3017-4f87-aa8a-5c9b94b6ef6e.png)

**Comment by aocenas** [comment]
Sorry but discussions are used specifically for discussions of feature requests of Grafana application. This seems like a very generic question and it also seems like not really about Grafana so I am going to close this.
